samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -170,6 +170,17 @@ The library requests this data automatically, but has no way to automatically re
 Therefore, all MQTT messages received from `nRF Cloud`_ must be be passed directly through to the :ref:`lib_nrf_cloud_agps` and :ref:`lib_nrf_cloud_pgps` libraries to be checked for A-GPS and P-GPS data, respectively.
 The :c:func:`location_assistance_data_handler` function handles this task, and MQTT messages are sent directly to it from the :file:`src/connection.c` file.
 
+This sample supports placing P-GPS data in external flash for the nRF9160 development kit version 1.0.1 and higher.
+Currently, you cannot combine this with full modem FOTA.
+To enable this, add the following parameter to your build command:
+
+``-DOVERLAY_CONFIG=overlay_pgps_ext_flash.conf``
+
+Also, specify your development kit version by appending it to the board name.
+For example, if your development kit version is 1.0.1, use the following board name in your build command:
+
+``nrf9160dk_nrf9160_ns@1_0_1``
+
```

Comment:
I might move this to after the note -- actually, the note box seems overkill to me, but I'll add rewriting it to my minor improvements ticket.